inc/class_parser.php

```diff
@@ -1746,6 +1746,13 @@ function text_parse_message($message, $options=array())
 		{
 			$this->options = $options;
 		}
+		else
+		{
+			foreach($options as $option_name => $option_value)
```

Comment:
Couldn't this just be `$this->options = array_merge($this->options, $options);`?

Reply:
I tried this first but it malfunctioned and didn't append the new key-value pair or override any old. That was odd but that happened repeatedly. So I placed it this way and it worked.

Reply:
Weird, this way works too anyway 😄